src/Tasks/BootstrapperUtil/Util.cs

```diff
@@ -75,6 +78,9 @@ public static CultureInfo GetCultureInfoFromString(string cultureName)
 
         public static CultureInfo DefaultCultureInfo => System.Threading.Thread.CurrentThread.CurrentUICulture;
 
+#if RUNTIME_TYPE_NETCORE
+        [SupportedOSPlatform("windows")]
```

Comment:
Same q here: is this a "class" thing or a "parts of it" thing?

Reply:
This one makes more sense to condition part of it. There are ~5 methods that don't need the platform guard so I figure we can split it here.

Reply:
If this wasn't here before, why does it need it now?

Reply:
because `DefaultPath` looks into the registry, which should only be allowed on windows.

another option could be to check if `IsWindows`, then return `GetCurrentDirectory()`